ceph/rados/core_workflows.py

```diff
@@ -2195,3 +2195,135 @@ def create_inconsistent_object(self, pool_name, object_name):
+        status, desc = self.get_daemon_status(daemon_type=daemon, daemon_id=id)
+        if int(status) != 1 or "running" not in desc:
+            log.error(
+                f"Input daemon {daemon}.{id} either does not exist or "
+                f"currently not running."
+            )
+            return False
+
+        # fetch daemon host
+        daemon_host = self.fetch_host_node(daemon_type=daemon, daemon_id=id)
+
+        # ensure crash module is enabled
+        self.node.shell(["ceph mgr module enable crash"])
+
+        # capture initial number of crash reports in the /var/lib/ceph/crash directory
+        _cmd = "cephadm shell -- ls /var/lib/ceph/crash/ | wc -l"
+        init_crash_reports, err = daemon_host.exec_command(cmd=_cmd, sudo=True)
+        # capture initial number of crash list from crash ls-new
+        init_crash_ls = len(self.run_ceph_command("ceph crash ls-new"))
+
+        # generate crash using ceph daemon command
+        asok_cmd = (
+            f"cephadm shell -- ceph daemon /var/run/ceph/ceph-{daemon}.{id}.asok assert"
+        )
+        try:
+            out, err = daemon_host.exec_command(cmd=asok_cmd, sudo=True)
+        except Exception as err_exec:
+            if "exception" not in str(err_exec):
+                log.error(f"STDERR: {err_exec}")
+                log.error("Could not crash the daemon with ceph daemon utility")
+                return False
+
+        crash_report_post, err = daemon_host.exec_command(cmd=_cmd, sudo=True)
+        if not crash_report_post > init_crash_reports:
+            log.error("New crash report directory not generated in /var/lib/ceph/crash")
+            return False
+        log.info("New crash report generated successfully")
+
+        # get the crash report directory
+        _cmd = "cephadm shell -- ls -rt /var/lib/ceph/crash/ | tail -1"
+        dir, err = daemon_host.exec_command(cmd=_cmd, sudo=True)
+        dir = str(dir).strip()
+
+        # copy the crash directory to tmp
+        fsid = self.run_ceph_command(cmd="ceph fsid")["fsid"]
+        _cmd = f"cp -r /var/lib/ceph/{fsid}/crash/'{dir}' /tmp/"
+        daemon_host.exec_command(cmd=_cmd, sudo=True)
+
+        if manual_inject:
+            # inject the crash into the cluster manually
+            self.configure_host_as_client(daemon_host)
+            inject_cmd = f"ceph crash post -i /tmp/{dir}/meta"
+            daemon_host.exec_command(sudo=True, cmd=inject_cmd)
+            if not warning_check(timeout=100):
+                log.error("Crash warning not found even after manual injection")
+                return False
+        else:
+            # wait for 500 secs for crash to be reported in ceph health
+            if not warning_check(timeout=500):
+                log.error("Crash warning was not generated automatically")
+
+        # ensure crash detail is populated in crash ls-new
+        crash_ls_post = self.run_ceph_command("ceph crash ls-new")
```

Comment:
Let's print all the command outputs in debug logs?

Reply:
done